File: app/src/main/java/com/yyide/chatim/activity/newnotice/fragment/NoticeReleaseAdapter.java
package com.yyide.chatim.activity.newnotice.fragment;

import com.chad.library.adapter.base.BaseQuickAdapter;
import com.chad.library.adapter.base.module.LoadMoreModule;
import com.chad.library.adapter.base.viewholder.BaseViewHolder;

import org.jetbrains.annotations.NotNull;

public class NoticeReleaseAdapter extends BaseQuickAdapter<String, BaseViewHolder> implements LoadMoreModule {
    public NoticeReleaseAdapter(int layoutResId) {
        super(layoutResId);
    }

    @Override
    protected void convert(@NotNull BaseViewHolder holder, String s) {

    }
}

File: app/src/main/java/com/yyide/chatim/activity/newnotice/fragment/NoticeReleaseAdapter.kt
package com.yyide.chatim.activity.newnotice.fragment

import com.chad.library.adapter.base.BaseQuickAdapter
import com.chad.library.adapter.base.module.LoadMoreModule
import com.chad.library.adapter.base.viewholder.BaseViewHolder

class NoticeReleaseAdapter(layoutResId: Int) : BaseQuickAdapter<String?, BaseViewHolder>(layoutResId), LoadMoreModule {
    override fun convert(holder: BaseViewHolder, s: String?) {}
}